game_event/views.py

```diff
@@ -1,6 +1,8 @@
 from .models import GameEvent
-from django.shortcuts import render, redirect
+from django.shortcuts import render, redirect, get_object_or_404
 from court.models import Court
+from game_event_player.models import GameEventPlayer
+from player.models import Player
```

Comment:
Please add the import statement for Player within the same line as BallGame instead of creating a new line in the imports.